src/plugins/messenger/MessengerPreview/components/MessengerImage/components/ImagePreview.tsx

```diff
@@ -0,0 +1,73 @@
+import { useMessangerImageContext } from '../MessangerImageContext';
+import { MessagePluginFactoryProps } from "../../../../../../common/interfaces/message-plugin";
+import { getBackgroundImage } from '../../../lib/css';
+
+export const getImagePreview = ({ React, styled }: MessagePluginFactoryProps) => {
+
+    const templateStyle = ({ template = 'media' }) => ({
+        media: {
+            paddingTop: "75%"
+        },
+        list: {
+            paddingTop: "50%"
+        },
+        generic: {
+            paddingTop: "50%",
+            display: "table",
+            width: "100%"
+        }
+    })[template];
+
+    const FixedImage = styled.div({
+        backgroundSize: "cover",
+        backgroundPosition: "center center",
+        cursor: 'pointer',
+        "&:hover": {
+            opacity: '.8'
+        },
+        "&:focus": {
+            opacity: '.8',
```

Comment:
I tested the focus style, I think we can leave it with the opacity instead of border or box-shadow. But also please add the same focus style for FlexImage